### PR TITLE
[bug 1194189] Fix scaling issue on iOS devices

### PR DESCRIPTION
**Pull request by rehandalal**
pretty simple.

@mythmon  r?

### CODE REVIEW

File: fjord/feedback/templates/feedback/base.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {% import 'includes/common.html' as common with context %}
 
 {% block extra_headers %}
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
```
Comment (willkg) [inline fjord/feedback/templates/feedback/base.html]: We only need this one. The other ones don't affect feedback forms. So we should ditch those changes from the PR.

Comment (willkg) [inline fjord/feedback/templates/feedback/base.html]: Scratch that. It won't affect the other things, so we should just leave it.

**Comment by mythmon** [comment]
The only device I have to test on is an iPad, which doesn't have the original problem. It doesn't try to zoom to the input field. The page looks the same before and after this patch. With this patch applied I can't zoom in or out though, which I guess is the point.

I'm good with this. r+ from me.

**Comment by willkg** [comment]
Thank you both! :)

**Pull request merged**